connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -234,6 +234,30 @@ public ConfigDef config() {
     @Override
     public org.apache.kafka.common.config.Config validate(Map<String, String> props) {
         List<ConfigValue> configValues = super.validate(props).configValues();
+        validateExactlyOnceConfigs(props, configValues);
+        validateEmitOffsetSyncConfigs(props, configValues);
+
+        return new org.apache.kafka.common.config.Config(configValues);
+    }
+
+    private static void validateEmitOffsetSyncConfigs(Map<String, String> props, List<ConfigValue> configValues) {
+        boolean offsetSyncsConfigured = configValues.stream()
+                .anyMatch(conf -> conf.name().startsWith(OFFSET_SYNCS_CLIENT_ROLE_PREFIX) || conf.name().startsWith(OFFSET_SYNCS_TOPIC_CONFIG_PREFIX));
+
+        if ("false".equals(props.get(EXACTLY_ONCE_SUPPORT_CONFIG)) && offsetSyncsConfigured) {
```

Comment:
Should it be `EMIT_OFFSET_SYNCS_ENABLED` rather than `EXACTLY_ONCE_SUPPORT_CONFIG` ?